### PR TITLE
[FEATURE] Mysql tls ssl support

### PR DESCRIPTION
**Pull request by jcortes**
### <samp>🤖 Generated by Copilot at dcca7db</samp>

Added SSL support to the `mysql` component actions and sources. Refactored some utility functions to improve readability and consistency. Updated the `mysql` component version and dependencies.

### <samp>🤖 Generated by Copilot at dcca7db</samp>

*  Simplify the function signatures of `getColumnProps` and `getColumnAndValueArrays` to accept a single `table` argument instead of two arguments `table` and `rejectUnauthorized` ([link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-764e6b5adcb58021f6f1488990148a4012c45cc90db9dfbd328426713dfbd4e5L1-R5),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-764e6b5adcb58021f6f1488990148a4012c45cc90db9dfbd328426713dfbd4e5L14-R21))
* Add the `ca`, `key`, and `cert` props to the `mysql` app and the `mysql` component actions and sources to allow the user to provide custom SSL certificates for secure connections to the MySQL server ([link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-e77cba66fc5b48199f70730a9b34b25ee7765a946a96cc60a31373cc4715c118L26-R51),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-7e47a681eba34dfb262ac30148c163e576c76844a8150e6f2cfecbad54895d12R36-R53),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-ab48419c4b605d8a7388c0e7cc414edc71d5433c4a6acc699efe608dae72fee3R36-R53),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-05c067593335f804114bddcb55e29af06d2d17c80fa19b6b0c76b49791c73c6bR17-R34),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-6fd75523adac6228be94eabdb6940eb6527d3fa0fe7468fd2bd6fe3b8561df4aR46-R63),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-379d5382cad77727c24f0c8f75f62215d60f018a6f3e4f78ffc10968b3374cffL38-R60),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-376db3b2ac940d9bb6d6487fa24de56835178eb93baa8bfaecad329ff92051fdL101-R124),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-b4729d8ea24d3f6e4bc55a0601de5cf4733a659ee6fbb5cf122cb9b50eb4501bR19-R36),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-9f49382e7c15082188244a49394bb0aea53e5fd5a511f6bcf230cdba619e1f28L24-R41),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-3a706b57a2df75b41bda5bafcd84723183a8b2e7cf95f658df55661af7b5a705R49-R52),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-12ad91445c4e18ed54f47aa18b71f41f2e3736833f852dd5752f8241050b7d3fL32-R53),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-b4172aae7e9de9502a0c5c9233d40b2799769ad73dcdd6e1e201b0fa1bf382f8L17-R31))
* Pass the `ca`, `key`, and `cert` props to the `mysql` app methods that create the `ssl` object and pass it to the `getConnection` method of the `mysqlClient` module ([link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-e77cba66fc5b48199f70730a9b34b25ee7765a946a96cc60a31373cc4715c118L46-R67),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-7e47a681eba34dfb262ac30148c163e576c76844a8150e6f2cfecbad54895d12R61-R63),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-ab48419c4b605d8a7388c0e7cc414edc71d5433c4a6acc699efe608dae72fee3R61-R63),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-05c067593335f804114bddcb55e29af06d2d17c80fa19b6b0c76b49791c73c6bR56-R58),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-6fd75523adac6228be94eabdb6940eb6527d3fa0fe7468fd2bd6fe3b8561df4aR72-R74),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-379d5382cad77727c24f0c8f75f62215d60f018a6f3e4f78ffc10968b3374cffR67-R69),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-376db3b2ac940d9bb6d6487fa24de56835178eb93baa8bfaecad329ff92051fdL118-R147),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-9f49382e7c15082188244a49394bb0aea53e5fd5a511f6bcf230cdba619e1f28L24-R41),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-3a706b57a2df75b41bda5bafcd84723183a8b2e7cf95f658df55661af7b5a705R49-R52),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-12ad91445c4e18ed54f47aa18b71f41f2e3736833f852dd5752f8241050b7d3fL32-R53),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-b4172aae7e9de9502a0c5c9233d40b2799769ad73dcdd6e1e201b0fa1bf382f8L17-R31))
* Pass the `ssl` object to the `executeQueryConnectionHandler` method of the `mysql` app that executes the SQL queries and handles the connection errors ([link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-7e47a681eba34dfb262ac30148c163e576c76844a8150e6f2cfecbad54895d12L63-R89),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-ab48419c4b605d8a7388c0e7cc414edc71d5433c4a6acc699efe608dae72fee3L63-R89),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-05c067593335f804114bddcb55e29af06d2d17c80fa19b6b0c76b49791c73c6bL43-R69),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-6fd75523adac6228be94eabdb6940eb6527d3fa0fe7468fd2bd6fe3b8561df4aL64-R90),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-379d5382cad77727c24f0c8f75f62215d60f018a6f3e4f78ffc10968b3374cffL76-R100),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-376db3b2ac940d9bb6d6487fa24de56835178eb93baa8bfaecad329ff92051fdL137-R170),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-3a706b57a2df75b41bda5bafcd84723183a8b2e7cf95f658df55661af7b5a705L69-R78),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-b4172aae7e9de9502a0c5c9233d40b2799769ad73dcdd6e1e201b0fa1bf382f8L25-R60))
* Change the signatures of several `mysql` app methods that interact with the MySQL server to accept an `args` object that contains the parameters and the `ssl` property ([link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-376db3b2ac940d9bb6d6487fa24de56835178eb93baa8bfaecad329ff92051fdL137-R170),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-376db3b2ac940d9bb6d6487fa24de56835178eb93baa8bfaecad329ff92051fdL154-R190),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-376db3b2ac940d9bb6d6487fa24de56835178eb93baa8bfaecad329ff92051fdL172-R291),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-376db3b2ac940d9bb6d6487fa24de56835178eb93baa8bfaecad329ff92051fdL245-R409))
* Update the versions of the `mysql` component actions and sources to reflect the changes made to the code and the props ([link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-e77cba66fc5b48199f70730a9b34b25ee7765a946a96cc60a31373cc4715c118L9-R9),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-7e47a681eba34dfb262ac30148c163e576c76844a8150e6f2cfecbad54895d12L8-R8),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-ab48419c4b605d8a7388c0e7cc414edc71d5433c4a6acc699efe608dae72fee3L8-R8),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-05c067593335f804114bddcb55e29af06d2d17c80fa19b6b0c76b49791c73c6bL8-R8),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-6fd75523adac6228be94eabdb6940eb6527d3fa0fe7468fd2bd6fe3b8561df4aL8-R8),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-379d5382cad77727c24f0c8f75f62215d60f018a6f3e4f78ffc10968b3374cffL9-R9),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-1b22bccadd8910a9c5ff70702aaca474b1a71e37753f441553a60d0439318c0fL3-R3),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-9f49382e7c15082188244a49394bb0aea53e5fd5a511f6bcf230cdba619e1f28L9-R9),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-21666d1a66b2853f520961e2f1f2b2571ab7a8ebf66e055982b39668424ff569L12-R12),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-3a706b57a2df75b41bda5bafcd84723183a8b2e7cf95f658df55661af7b5a705L11-R11),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-12ad91445c4e18ed54f47aa18b71f41f2e3736833f852dd5752f8241050b7d3fL11-R11),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-b4172aae7e9de9502a0c5c9233d40b2799769ad73dcdd6e1e201b0fa1bf382f8L9-R9))
* Import the `fs` module from Node.js in the `mysql` app to read the custom SSL certificate files from the file system ([link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-376db3b2ac940d9bb6d6487fa24de56835178eb93baa8bfaecad329ff92051fdR2))
* Add the `components/zoho_analytics`, `components/zoho_catalyst`, and `components/zoho_sign` entries to the `pnpm-lock.yaml` file, which are related to other components that were added or updated in the same pull request, but are not directly related to the `mysql` component changes ([link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbR5615-R5620),[link](https://github.com/PipedreamHQ/pipedream/pull/7403/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbR5676-R5678))

### CODE REVIEW

**Review by vellames — DISMISSED** [review]
lgtm

File: components/mysql/actions/create-row/create-row.mjs [hunk 1]
```diff
@@ -6,7 +6,7 @@ export default {
   name: "Create Row",
   description: "Adds a new row. [See the docs here](https://dev.mysql.com/doc/refman/8.0/en/insert.html)",
   type: "action",
-  version: "0.0.4",
+  version: "0.0.10",
```
Comment (michelle0927) [inline components/mysql/actions/create-row/create-row.mjs:9]:
```suggestion
  version: "0.0.5",
```

**Review by michelle0927 — COMMENTED** [review]
LGTM! Just one minor comment. Ready for QA.

**Review by GTFalcao — DISMISSED** [review]
Hi @jcortes , thanks for the contribution, looks good to me!

**Review by GTFalcao — APPROVED** [review]

**Pull request merged**